lib/manageiq/cross_repo/runner/base.rb

```diff
@@ -81,10 +81,10 @@ def ci_config
       def defaults
         @defaults ||= {
           "node_js" => {
-            "language" => "node_js",
-            "node_js"  => ["18"],
-            "install"  => "npm install",
-            "script"   => "npm test"
+            "language"      => "node_js",
+            "node_js"       => ["18"],
+            "before_script" => "yarn",
```

Comment:
NOTE we are parsing "Prepare tests" as "before_script" which for ui-service runs `bin/setup`.  The issue was we were running `install` AND `before_script` which was causing the failure because npm install was failing and not the intended way to do the install.